.changeset/polite-snails-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/math-input": minor
+---
+
+Updating tabbar experience to use arrow-key navigation to access the other tabitems.
```

Comment:
This comment is typically echoed in release notes, etc. So, including the widget name somewhere in the description might be helpful.